website/www/site/content/en/documentation/ml/overview.md

```diff
@@ -59,3 +59,4 @@ In order to automate and track the AI/ML workflows throughout your project, you
 
 You can find examples of end-to-end AI/ML pipelines for several use cases:
```

Comment:
Please add more descriptions to this list, for each item a small paragraph of what this does with high level example of the use case shown .

Reply:
@Juta please have a look and let's discuss this

Reply:
@damccorm I will also prefer to do in a different PR. @Juta what do you think?

Reply:
SGTM, I'll go ahead and merge and we can circle back to this

Reply:
Yes ok let's pick this up in a next PR